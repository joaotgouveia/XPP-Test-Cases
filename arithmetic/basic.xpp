1 + 1;;

1 + true;;

true + 1;;

true + true;;

1 + { a = 1 };;

{ a = 1 } + 1;;

{ a = 1 }.a + 1;;

{ a = 1; b = 2 }.b + 1;;
