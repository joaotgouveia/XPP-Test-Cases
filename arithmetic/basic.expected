Value: 2
Type: int

throws TypeCheckError

throws TypeCheckError

throws TypeCheckError

throws TypeCheckError

throws TypeCheckError

Value: 2
Type: int

Value: 3
Type: int
